src/modules/Dashboard/components/ActivityGrid/ActivityGrid.tsx

```diff
@@ -2,22 +2,26 @@ import { useState } from 'react';
 import { useTranslation } from 'react-i18next';
 import { generatePath, useNavigate, useParams } from 'react-router-dom';
 
-import { ButtonWithMenu, Spinner, Svg, EmptyState } from 'shared/components';
+import { ButtonWithMenu, EmptyState, Spinner, Svg } from 'shared/components';
 import { StyledBody, StyledFlexTopCenter, StyledFlexWrap } from 'shared/styles';
 import { page } from 'resources';
 import { ActivitySummaryCard } from 'modules/Dashboard/components';
+import { workspaces } from 'shared/state';
 
 import { StyledButton } from './ActivityGrid.styles';
 import { ActivityGridProps } from './ActivityGrid.types';
 
 export const ActivityGrid = ({
-  rows,
   isLoading = false,
-  'data-testid': dataTestid,
+  rows,
+  TakeNowModal,
+  'data-testid': dataTestId,
```

Comment:
Nit: While I get the rationale for uppercasing the `I`, FWIW, the lowercase `i` version is actually much more common in the codebase at the moment. I suppose the basis for the lowercase `i` choice is it being the result of a pure kebab-case to camelCase conversion. But I agree it's a bit odd to have this be the only identifier in the codebase where we consistently do not camelCase `…Id`, so if that's the consensus, I'm fine with uppercasing it from now on.

Reply:
I would like to opt for uppercasing the I from now on, unless there's a good reason for sticking with the current convention. But for this PR, I'm going to change it back

Reply:
Actually, I'm gonna do a 180 here and keep the uppercase I. I see that Carlos has already started this change, and I don't want to go backwards